Makefile

```diff
@@ -0,0 +1,115 @@
+.EXPORT_ALL_VARIABLES:
```

Comment:
Please add some documentation that may help someone get up and running the with `make`, as well as any other topical commentary, to https://github.com/commontoolsinc/system/blob/main/DEVELOPMENT.md

Reply:
more than happy to! didn't want to prematurely spend time on docs until I'd had your eyes on it.

Reply:
- https://github.com/commontoolsinc/system/issues/67